plugins/network-elements/nuage-vsp/src/com/cloud/network/element/NuageVspElement.java

```diff
@@ -483,7 +663,7 @@ protected HostVO getNuageVspHost(Long physicalNetworkId) throws CloudException {
             nuageVspHost = _hostDao.findById(config.getHostId());
             _hostDao.loadDetails(nuageVspHost);
         } else {
-            throw new CloudException("Nuage VSD is not configured on physical network " + physicalNetworkId);
+            throw new CloudRuntimeException("Nuage VSD is not configured on physical network " + physicalNetworkId);
```

Comment:
Changed code to <code>throw new CloudRuntimeException("There is not Nuage VSP device configured on physical network " + physicalNetworkId)</code>. We can't identify about the Nuage VSP device that failed since there is none configured.